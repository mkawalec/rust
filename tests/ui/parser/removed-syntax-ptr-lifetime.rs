type bptr = &lifetime/isize; //~ ERROR expected one of `!`, `(`, `::`, `;`, `<`, `is`, or `where`, found `/`
