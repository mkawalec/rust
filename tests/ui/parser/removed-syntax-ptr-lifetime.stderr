error: expected one of `!`, `(`, `::`, `;`, `<`, `is`, or `where`, found `/`
  --> $DIR/removed-syntax-ptr-lifetime.rs:1:22
   |
LL | type bptr = &lifetime/isize;
   |                      ^ expected one of 7 possible tokens

error: aborting due to previous error

